docker-compose.yml

```diff
@@ -19,38 +18,39 @@ services:
       # https://docs.docker.com/engine/reference/commandline/run/
       - ./db:/data/db:Z
 
-  sdx-controller:
+  sdx-controller-service:
     image: sdx-controller
     container_name: sdx-controller
+    depends_on:
+      - sdx-controller-mongodb-service
     tty: true
     build: ./
     ports:
       - 8080:8080
     environment:
-      # Use sdx-controller-mongodb service specified above.
```

Comment:
Are the env vars being moved to .env file?

Reply:
Yes. However MongoDB connection would be using Docker Compose networking, so sdx-controller will be connecting to the `sdx-controller-mongodb` container.  No need of making that an environment variable.

Reply:
I mean, remove all the `- XX='xxx'`, and replace with something like:
```
environment:
- .env
```

Reply:
To be clear, I am not too opposed to just setting `env_file`. That perhaps is simpler, and the other details are not worth the worry? It probably is just a matter of opinion?

I am not a Docker expert, so my opinion here isn't worth much. :-)